src/index.js

```diff
@@ -1,8 +1,7 @@
 import './style.css';
 import DoTask from './task.js';
 import './delete.png';
-// eslint-disable-next-line no-unused-vars
-import action from './action.png';
+import './action.png';
```

Comment:
- [ ] idly solve this bug when refreshing the page should not remove all of the  completed tasks

https://drive.google.com/file/d/1SsEx3pwIX8R05hblO3fHduGx-GBd9EvH/view?usp=sharing

- [ ] also according to the design pattern all completed tasks should have lines through them.

<img width="508" alt="image" src="https://user-images.githubusercontent.com/65068771/222606705-a17788dc-f271-48fc-8ae0-7fe1d9128427.png">